backend/app/middlewares/loginAdminUser.js

```diff
@@ -0,0 +1,33 @@
+const authJwt = require("./authJwt");
+
+isAdmin = (req, res, next) => {
+  User.findById(req.userId).exec((err, user) => {
```

Comment:
You can replace the `User.findById` with `User.findOne({
    username: req.body.username
  })` because we know the username but not the userId at this point.